hack/demo/deploy/alibaba-provider.yaml

```diff
@@ -16,7 +16,7 @@ spec:
         control-plane: provider-alibaba
     spec:
       containers:
-      - image: crossplane/provider-alibaba:v1
+      - image: crossplane/provider-alibaba:v0.0.1
```

Comment:
In practice this is not how provider will be installed with Crossplane, and so this `Deployment` image will not be used. However, if you do want to keep this around for a demo then I would suggest potentially just using the `master` tag.